### PR TITLE
release raiwidgets 0.9.0

### PR DESCRIPTION
**Pull request by imatiach-msft**
## v0.9.0

- educational materials
  - add Model Analysis notebook for regression
- new features
  - local importance chart for counterfactual dashboard
  - add multi causal policy support
  - add delete tab dialog and remove inline widget for model assessment dashboard
  - add treatment policy and table to causal analysis dashboard
  - create new custom individual feature importance component
- breaking changes
- bug fixes
  - explicitly pass model task to MimicExplainer in model analyzer
  - validate treatment features in causal manager
  - Model Analysis:
    - sticky layer for cohort and dashboard settings
    - restrict EA tree to 80% width
    - restrict ipykernel<0.6
    - fix flyout title and description
    - fix add button behavior by adding components in the appropriate spot
    - show add button only for components that can render
  - Error Analysis:
    - remove 100k row limit
    - fix zero nodes in tree view
    - fix search functionality in features list
    - fix displayed filter order in tree view to start from root node
    - raise warning for older version of pandas when feature names contain '-'
    - expand number of decimals shown in filter tooltips when values below 0.1
    - fix regression case for 1d heatmap matrix metrics being calculated incorrectly
  - set original data from dropdown for counterfactuals
  - use test data instead of train data for error analysis manager in model analyzer
  - causal analysis to keep all levels in dataframe outputs
  - pin networkx pip package to prevent installation of matplotlib
  - add target_column causal policy gain to UI control
- other
  - unit tests for multiclass classification for causal analysis
  - add validations for input parameters for Model Analysis class
  - add model analysis notebooks to notebook gate
  - add test for causal manager for None categoricals
  - fix example code for model assessment

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,47 @@ this file to understand what changed.
...
+    - expand number of decimals shown in filter tooltips when values below 0.1
+    - fix regression case for 1d heatmap matrix metrics being calculated incorrectly
+  - set original data from dropdown for counterfactuals
+  - use test data instead of train data for error analysis manager in model analyzer
+  - causal analysis to keep all levels in dataframe outputs
+  - pin networkx pip package to prevent installation of matplotlib
+  - add target_column causal policy gain to UI control
+- other
+  - unit tests for multiclass classification for causal analysis
+  - add validations for input parameters for Model Analysis class
```
Comment (gaugup) [inline CHANGES.md:63]: Shouldn't the validation bit go into the ModelAnalysis section?

Comment (gaugup) [reply CHANGES.md:63]: Perhaps missing set of of changes here:-

https://github.com/microsoft/responsible-ai-widgets/pull/695
https://github.com/microsoft/responsible-ai-widgets/pull/699
https://github.com/microsoft/responsible-ai-widgets/pull/686

**Review by gaugup — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (imatiach-msft) [reply CHANGES.md:63]: they are already there, but I moved the first two under "Model Analysis" bug fix section (at end) and the last one is already under "Error Analysis" section ("raise warning for older version of pandas when feature names contain '-'")

File: responsibleai/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 dice-ml>=0.6.1,<0.7
 econml==0.12.0b4
-erroranalysis>=0.1.8
+erroranalysis>=0.1.11
 interpret-community>=0.18.1
```
Comment (gaugup) [inline responsibleai/requirements.txt:4]: Do we need to upgrade interpret-community to 0.19.x?

**Review by gaugup — APPROVED** [review]

File: responsibleai/requirements.txt (hunk 2 above)
Comment (imatiach-msft) [reply responsibleai/requirements.txt:4]: I wanted to hold off on that since it's so new in case it causes any environment conflicts I can't foresee
